lib/main.dart

```diff
@@ -48,6 +52,7 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   final cardStream = supabase.from('card').stream(primaryKey: ['id']);
   bool isGridView = false;
+  final cardBox = Hive.box('myBox');
```

Comment:
This is a good start, but we need to label keys correctly here. Calle it  ```final cardBox = Hive.box('card');``` instead of  ```final cardBox = Hive.box('myBox');```